.github/workflows/kitchen.yml

```diff
@@ -146,6 +146,10 @@ jobs:
       FORCE_FFI_YAJL: ext
       CHEF_LICENSE: accept-no-persist
     steps:
+      - name: Update CentOS-8 Repositories
+        if: matrix.os == 'centos-8'
+        run: |
+          sed -ibak -e 's!#baseurl=http://mirror.centos.org/$contentdir!baseurl=http://vault.centos.org!g' CentOS-Stream-*repo
```

Comment:
This runs on the host not the testing container so this would actually run on the ubuntu 22.04 instance spun up by github actions. You need to actually add that said line to the kitchen.yml file here https://github.com/chef/chef/blob/chef-17/kitchen-tests/kitchen.yml#L103 to have the container that comes up run that command before doing anything else.

Reply:
oh rats. thanks for catching that...